src/renault_api/model/kamereon.py

```diff
@@ -92,18 +92,95 @@ class KamereonPersonResponse(KamereonResponse):
 )()
 
 
+class EnergyCode(Enum):
+    """Enum for vehicleDetails energy code."""
+
+    ESSENCE = "ESS"
+    ELECTRIQUE = "ELEC"
```

Comment:
In english

Reply:
Addressed in PR #68 : https://github.com/hacf-fr/renault-api/pull/68/commits/28c27b9028158d3c5be1ccc7d9259481ef7be6c0